workers/unity/Packages/io.improbable.gdk.core/Subscriptions/CallbackManagers/GuardedCallbackManagerSet.cs

```diff
@@ -33,15 +34,14 @@ public List<TManager> GetManagers()
 
                 indexesToAdd.Clear();
             }
-
-            return callbackManagers;
         }
 
-        public void InvokeCallbacks()
+        public void InvokeEach(Action<TManager> callback)
```

Comment:
out of interest, why change this?

Reply:
This removes the requirement for having 2 type of GuardedCallbackSet's, and instead have a generic version.